apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -159,7 +167,12 @@ suite('Settings >', function() {
         'operatorResource.sms.maxConcat',
         10
       );
+      triggerSettingsReqSuccess('messages.mms.sendReadReport.enabled', true);
+
+      Settings.sendReadReport.then((result) => assert.equal(result, true)
+        ).then(done, done);
 
+      sinon.assert.calledWith(Settings.initSendReadReport, true);
```

Comment:
yeah, But i think it's good to keep it as previous assertion check the promise return value and this one check that function is called with expected argument. Though it means the same.
Still, if you say, it will remove it then .  @julienw